Neuro/Vision/VisionHandler.cs

```diff
@@ -247,4 +247,9 @@ private static string GetLocationFromPosition(Vector2 position)
         // We're not in an actual room, so say which room we're nearest to
         return nearPrefix + TranslationController.Instance.GetString(closestLocation.RoomId);
     }
+
+    public Dictionary<PlayerControl, LastSeenPlayer> GetPlayerLocations()
+    {
+        return _playerLocations;
```

Comment:
This can be returned as an `IReadOnlyDictionary<>` to convey that only the vision handler should be modifying the dictionary.